Models/models.js

```diff
@@ -9,13 +9,28 @@ exports.selectCategories = () => {
     })
 }
 
-exports.selectReviews = () => {
-    return db.query(
+exports.selectReviews = (order = 'DESC', sort_by = 'created_at', category) => {
+    if (!['ASC', 'DESC'].includes(order.toUpperCase())){
+        order = 'DESC'
```

Comment:
do you actually want to change order to desc here? or would you want to throw an error if order is not asc or desc?